### PR TITLE
Add videocalls to livechat

### LINKED ISSUE #3685: Ad-hoc audio/video Livechat feature suggestion
It would be really helpful for educational contexts to be able to switch from text-only Livechats to audio and/or video Livechats.

Thanks for considering this request.
Paul

### PR DESCRIPTION
**Pull request by sampaiodiego**
@RocketChat/core 

Closes #3685

And also:
- Few improvements to video tabBar
- Add translations to action links

If enabled, it will add a new button on the livechat register screen:
image

And a button on the chat screen:
image

Once the videocall is requested, it will start a jitsi video conference and the guest will be waiting for someone to join:
image

Also a message will be sent to the agent accept it or not:
image

If accepted, the will join the call.. If declined, the conversation will be closed.

### CODE REVIEW

**Comment by sampaiodiego** [comment]
Ready for review and merge!

File: packages/rocketchat-livechat/app/client/lib/LivechatVideoCall.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+		if (typeof JitsiMeetExternalAPI === 'undefined') {
+			$.getScript('/packages/rocketchat_videobridge/client/public/external_api.js');
+		}
+	}
+
+	askPermissions(callback) {
+		if (Meteor.isCordova) {
+			cordova.plugins.diagnostic.requestCameraAuthorization(() => {
+				cordova.plugins.diagnostic.requestMicrophoneAuthorization(() => {
```
Comment (rodrigok) [inline packages/rocketchat-livechat/app/client/lib/LivechatVideoCall.js]: Will this work with our current mobile apps or will generate an exception?

Comment (sampaiodiego) [inline packages/rocketchat-livechat/app/client/lib/LivechatVideoCall.js]: hmm.. actually, this code does not run on our mobile apps (there is no way to access the route `/livechat` from within the app).

so yeah, this code is useless =)

**Comment by geekgonecrazy** [comment]
This is really cool 👍

**Pull request merged**